content/blog/2022/07/camel-quarkus-release-2.7.2/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Camel Quarkus 2.7.2 Released"
+date: 2022-07-04
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 2.7.2 Released"
+summary: "Camel Quarkus 2.7.2 brings Quarkus 2.7.6.Final, Camel 3.14.4"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the 2.7.2 release of Camel Quarkus.
+It brings Camel 3.14.4, Quarkus 2.7.6.Final and several bug fixes.
+Please check the [Release notes](/releases/camel-quarkus/release-2.7.2/)
```

Comment:
Did you mean?

```suggestion
Please check the [Release notes](/releases/q-2.7.2/)
```

Reply:
Yeah, thanks for the correction, Zoran. I was not sure at all.